### PR TITLE
Fix test class inheritance and absolute paths

### LINKED ISSUE #1: Importer does not catch tests from class interitance
In my test suite I have some base classes that define tests for similarily structured tests.

Example:

```php
abstract class BasePaymentMethodCest
{
    public function testCreatePaymentMethod(AcceptanceJsTester $I)
    {
        // ...
    }
    public function testShopOrderWithPaymentMethod(AcceptanceJsTester $I)
    {
        // ...
    }
}
abstract class StripePaymentMethodCest extends BasePaymentMethodCest
{
    // ... contains only config, no test functions
}
class StripeCreditCardPaymentMethodCest extends StripePaymentMethodCest
{
    // ... contains only config, no test functions
}
```

`testCreatePaymentMethod` and `testShopOrderWithPaymentMethod` are not detected by the importer, so `StripeCreditCardPaymentMethodCest` does not show up in Testomat.

### LINKED ISSUE #2: Trimming absolute path does not work
Here:

https://github.com/testomatio/php-list-tests/blob/57087346b07e845d189f49d9837c2de7c41b1852/src/CheckTests.php#L64-L66

`$this` does not refer to the TestData class but to CheckTests class so file does never get updated.

I have a fix, will send a PR.

### PR DESCRIPTION
**Pull request by cebe**
Fixes #1 and fixes #2

### CODE REVIEW

File: src/TestData.php [hunk 1]
```diff
@@ -75,12 +75,11 @@ private function humanizeClass($name)
         return ucfirst($name);
     }
 
-    public function update(\Closure $closure)
+    public function removeAbsolutePath($absolutePath)
     {
-        $closure->call($this);
+        $this->file = ltrim(preg_replace('~^'. preg_quote($absolutePath, '~') . '~', '', $this->file), DIRECTORY_SEPARATOR);
```
Comment (cebe) [inline src/TestData.php:80]: probably better implementation:

```suggestion
        if (str_starts_with($this->file, $absolutePath)) {
            $this->file = ltrim(substr($this->file, strlen(absolutePath)), DIRECTORY_SEPARATOR);
        }
```

**Comment by cebe** [comment]
Tests are broken because Reflection can not find `Codeception\Test\Unit` class... not sure how to fix that.

**Comment by DavertMik** [comment]
Thanks! I will take it from this point

**Comment by DavertMik** [comment]
@cebe Thanks! I fixed the autoloading of additional classes. Had to add 2 more reflection sources. Please update to 0.1.8 and check if it works for you.

I remember I couldn't get it working for nested classes but now with your help it actually works!

**Pull request merged**